src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SearchResults from "../components/SearchResults";
+
+describe("Search Results", () => {
+  it("renders correctly", () => {
+    const { asFragment } = render(<SearchResults results={[]} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+  it("renders with results", () => {
+    const results = ["search-image1", "search-image2", "search-image3"];
+    const { asFragment } = render(<SearchResults results={results} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+  it("renders no results message when there are no results", () => {
+    render(<SearchResults results={[]} hasSearched={true} />);
+    const noResultsMessage = screen.getByText(
+      "No results" && "Please try another search"
+    );
+    expect(noResultsMessage).toBeInTheDocument();
+  });
+  it("renders images when there are results to return", () => {
+    const results = [
+      "search-image1",
+      "search-image2",
+      "search-image3",
+      "search-image4]",
+    ];
+    const hasSearched = true;
+    render(<SearchResults results={results} hasSearched={hasSearched} />);
+    const imageResults = screen.queryAllByAltText("spaceImage");
+    expect(imageResults.length).toBe(4);
```

Comment:
Slight formatting tweak, you don tneed to declare a variable here for the boolean, you can just pass it in. It makes sense to keep the results array as a variable due to its size and to make the test clearer
```
    render(<SearchResults results={results} hasSearched={true} />);
    const imageResults = screen.queryAllByAltText("spaceImage");

    expect(imageResults.length).toBe(4);
```
Also, if you get around to updating the alt text to something more meaningful as described above, you should update this test to accept this new info and keep it nice and accurate